superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/BigNumberViz.tsx

```diff
@@ -26,7 +26,7 @@ import {
   computeMaxFontSize,
   BRAND_COLOR,
   styled,
-  QueryObjectFilterClause,
+  BinaryQueryObjectFilterClause,
```

Comment:
I think yes, because `DrillDetailTableControls` is currently only set up to correctly display binary filters (e.g. column = value), and if you change the type to `QueryObjectFilterClause` there you'll get typing errors that you can't resolve without adding new logic for displaying the other types.  Set filters and unary filters are definitely possible to add display support for in the future but in its current state it only works with binary filters.